.env.example

```
@@ -1 +1,5 @@
-CHEC_PUBLIC_KEY=""
\ No newline at end of file
+CHEC_PUBLIC_KEY=
+CHEC_API_URL=api.chec.io
```

even though this is just the example it'll still most likely be copied and pasted into the `.env`. Not sure if the CLI cares if about the http protocol though.
```suggestion
CHEC_API_URL=https://api.chec.io
```

---

It works either way, but John's suggested edit is better because it's actually a URL